package po;

import java.io.Serializable;
import java.math.BigDecimal;

public class Book implements Serializable {
	private Integer id;
	private String title;
	private BigDecimal price;
	private Integer stock;

	public Book() {
	}

	public Book(Integer id, String title, BigDecimal price, Integer stock) {
		this.id = id;
		this.title = title;
		this.price = price;
		this.stock = stock;
	}

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public BigDecimal getPrice() {
		return price;
	}

	public void setPrice(BigDecimal price) {
		this.price = price;
	}

	public Integer getStock() {
		return stock;
	}

	public void setStock(Integer stock) {
		this.stock = stock;
	}
}
